com.unity.render-pipelines.universal/Shaders/2D/Light2D-Point-Volumetric.shader

```diff
@@ -29,15 +29,15 @@ Shader "Hidden/Light2d-Point-Volumetric"
             struct Varyings
             {
                 float4  positionCS      : SV_POSITION;
-                float2  uv              : TEXCOORD0;
-                float2	screenUV        : TEXCOORD1;
-                float2	lookupUV        : TEXCOORD2;  // This is used for light relative direction
-                float2	lookupNoRotUV   : TEXCOORD3;  // This is used for screen relative direction of a light
+                half2   uv              : TEXCOORD0;
```

Comment:
if you have a texture with more than 1024x1024 you don't have enough precision to store in uv in half.